modules/UI/side_pannels/contactlist/ContactListView.js

```diff
@@ -69,10 +79,19 @@ function getContactEl (id) {
  * Contact list.
  */
 var ContactListView = {
-    init (model) {
-        this.model = model;
+    init () {
+        initHTML();
         this.lockKey = 'roomLocked';
         this.unlockKey = 'roomUnlocked';
+    },
+
+    /**
+     * setup ContactList Model into ContactList View
+     *
+     * @param model
+     */
+    setup (model) {
```

Comment:
create separate method for setup model. We use this methods in different parts of application